packages/core/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: core
 description: Core components for Widgetbook Demo App
-version: 0.1.0+1
+version: 0.0.1
```

Comment:
Ok looks like we downgraded the version here which explains the changelog. Is this necessary? Can we put the version to `0.1.1` instead?

Reply:
Fixed 👍